src/plans/details/tabs/VirtualMachines/components/MigrationStatusVirtualMachineList/MigrationStatusVirtualMachinesList.tsx

```diff
@@ -68,16 +67,15 @@ const MigrationStatusVirtualMachinesList: FC<MigrationStatusVirtualMachinesListP
     <TableSortContextProvider fields={planMigrationVirtualMachinesFields}>
       <StandardPageWithSelection<MigrationStatusVirtualMachinePageData>
         canSelect={canSelect}
-        CellMapper={MigrationStatusVirtualMachinesRow}
+        cell={MigrationStatusVirtualMachinesRow}
         dataSource={dataSource}
-        ExpandedComponent={MigrationStatusExpandedPage}
+        expanded={MigrationStatusExpandedPage}
         expandedIds={expandedIds}
         fieldsMetadata={planMigrationVirtualMachinesFields}
         GlobalActionToolbarItems={actions}
         namespace={''}
```

Comment:
can we have this as default value in the component rather than calling a prop with empty value?